nuxeo-aggregation/nuxeo-dropdown-aggregation.html

```diff
@@ -25,6 +25,8 @@
       :host {
         display: block;
         width: 100%;
+        height: 100%;
+        overflow: auto;
```

Comment:
Adding this at the selectivity level, like what was done on master, doesn't help? Or does it cause side effects?

Reply:
If we apply it selectivity it doesn't make the trick, the parent needs to have 100% of its parent and the overflow needs to happen on the nuxeo-dropdown-aggregation.

Reply:
But on master we did the overflow on selectivity, right?

Reply:
Yes, on master we did it and, after checking it again, we can do it as well here. Going to change it and push it again. Thanks!

Reply:
I'm more concerned about the consistency of the solution than where it is actually located! :P